README.md

```diff
@@ -14,7 +14,7 @@ Ensure that the prettier program is installed:
 which prettier
 ```
 
-If prettier is not installed already, you can install prettier using `npm -g prettier` or via your package manager.
+If prettier is not installed already, you can install prettier using `npm install -g prettier` or via your package manager.
```

Comment:
Ah, thanks for the catch!

I was the contributor who added this line of documentation, so I take full blame :)